modules/Bio/EnsEMBL/Compara/Utils/JIRA.pm

```diff
@@ -644,10 +653,17 @@ sub _http_request {
     $content_data //= {};
     # Create the HTTP request and LWP objects to get the response for the given arguments
     $self->{_logger}->debug("$method Request on $url\n");
-    my $request = HTTP::Request->new($method, $url);
-    $request->authorization_basic($self->{_user}, $self->{_password});
-    # The content data will be sent in JSON format
-    $request->header('Content-Type' => 'application/json');
+    my $request;
+    if ( $self->{_auth_token} ) {
```

Comment:
```suggestion
    if ( exists $self->{_auth_token} ) {
```

Reply:
I don't really understand what this is meant to catch.. Cases where `$self->{_auth_token}` is 0?

Reply:
Unless my understanding is off (which is possible), you not always have `$self->{_auth_token}` defined, so you need to check that `_auth_token` key exists in the hash before accessing it.

Reply:
Nope, perl can handle that and will return `undef` in that case. Also, I'm sure I would have seen some failures in testing if this were the case. I tested both cases where I have the env set and don't have the env set and it worked fine.